cli/send.go

```diff
@@ -167,6 +188,8 @@ var SendCmd = &cli.Command{
 			params.Method = abi.MethodNum(cctx.Uint64("method"))
 		}
 
+		_, _ = fmt.Fprintf(cctx.App.Writer, "Sending message from: %s\nSending message to: %s\nUsing Method: %d\n", params.From.String(), params.To.String(), params.Method)
```

Comment:
failing unit test because of this now, see CI

```
        	Error Trace:	/home/runner/work/lotus/lotus/cli/send_test.go:115
        	Error:      	Not equal: 
        	            	expected: "bafy2bzacecno6zh4xdnhzugo6fnenefhbmoia3fw72wbqnn55jqoov7xbnob4\n"
        	            	actual  : "Sending message from: f410faaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaonc6iji\nSending message to: f01\nUsing Method: 3844450837\nbafy2bzacecno6zh4xdnhzugo6fnenefhbmoia3fw72wbqnn55jqoov7xbnob4\n"
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1,4 @@
        	            	+Sending message from: f410faaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaonc6iji
        	            	+Sending message to: f01
        	            	+Using Method: 3844450837
        	            	 bafy2bzacecno6zh4xdnhzugo6fnenefhbmoia3fw72wbqnn55jqoov7xbnob4
        	Test:       	TestSendEthereum/simple
```

Reply:
Fixed.